openapi-python-templates/travis.mustache

```diff
@@ -0,0 +1,9 @@
+# ref: https://docs.travis-ci.com/user/languages/python
```

Comment:
Isn't this a very opinionated generation? Why not CircleCI or Gitlab? I would personally leave this out...

Reply:
It's from standard python client generator. I'll additionally check whether this is necessary

Reply:
Oh I see, well I guess it doesn't hurt to have this, probably better than nothing and might even spark inspiration for other collaborators to provide further implementations.